### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by dzenanz**
Logic is: it is easier to remove this, then to add. What should be added is less intuitive then what should be removed.

### CODE REVIEW

File: SlicerMONAIViz.py [hunk 1]
```diff
@@ -438,7 +438,16 @@ def onAddTransform(self):
 
         t = self.ui.transformsComboBox.currentText
         m = self.ui.modulesComboBox.currentText
-        self.addTransform(-1, None, t, None)
+
+        v = ""
+        if t[-1] == "d":  # this is a dictionary transform
+            # now exclude some transforms whose name happens to end with d
```
Comment (SachidanandAlle) [inline SlicerMONAIViz.py:444]: I think list transform has this flag to check if it's a dictionary transform

Comment (SachidanandAlle) [reply SlicerMONAIViz.py:444]: https://github.com/Project-MONAI/SlicerMONAIViz/blob/main/SlicerMONAIVizLib/utils.py#L46

Comment (dzenanz) [reply SlicerMONAIViz.py:444]: Thank you for the suggestion. I just searched, and I could not find a `isDictionary()` or something similar.

I checked `Allow edits by maintainers`. Please feel free to amend the commit, or add a new commit which does this better.

Comment (dzenanz) [reply SlicerMONAIViz.py:444]: I now see https://github.com/Project-MONAI/SlicerMONAIViz/blob/main/SlicerMONAIVizLib/utils.py#L46. I am still unsure how to use it properly.

**Comment by dzenanz** [comment]
Force push does a plain rebase.

**Comment by dzenanz** [comment]
I just did another plain rebase.

File: SlicerMONAIViz.py (hunk 1 above)
Comment (wyli) [reply SlicerMONAIViz.py]: thanks, Sachi is out of office today, I'm merging this for now and we can follow up on the dictionary types.

**Review by SachidanandAlle — APPROVED** [review]

File: SlicerMONAIViz.py (hunk 1 above)
Comment (dzenanz) [reply SlicerMONAIViz.py]: I created #15 to track this.

**Pull request merged**